src/WebJobs.Script/Workers/EnvironmentCondition.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
the number of conditions might grow in future so it might be worth create a new directory such as `/Profiles/Conditions`